rpi-clone

```diff
@@ -27,18 +27,20 @@ then
     exit 1
 fi
 
-raspbian=0
-raspbian_buster=0
-if [ -f /etc/os-release ]
+rpios=0
+rpios_recent=0
+if [ -f /etc/os-release ] && [ -f /proc/device-tree/model ]
 then
 	pretty=`cat /etc/os-release | grep PRETTY`
-	if [[ "$pretty" == *"Raspbian"* ]]
+	osversion=`cat /etc/os-release | grep VERSION_ID | cut -d'"' -f2`
+	model=`tr -d '\0' < /proc/device-tree/model`
+	if [[ "$pretty" == *"Raspbian"* || ("$model" == *"Raspberry"* && "$pretty" == *"Debian"*) ]]
 	then
-		raspbian=1
+		rpios=1
 	fi
-	if ((raspbian)) && [[ "$pretty" == *"buster"* ]]
+	if ((rpios)) && ((osversion >= 10))
```

Comment:
I like this check for the osversion :+1:

Reply:
Thanks, normally, it should work with later versions of Debian.